br/pkg/lightning/mydump/loader_test.go

```diff
@@ -622,3 +622,82 @@ func (s *testMydumpLoaderSuite) TestFileRouting(c *C) {
 		},
 	})
 }
+
+func (s *testMydumpLoaderSuite) TestInputWithSpecialChars(c *C) {
+	/*
+		Path/
+			test-schema-create.sql
+			test.t%22-schema.sql
+			test.t%22.0.sql
+			test.t%2522-schema.sql
+			test.t%2522.0.csv
+
+			db%22.t%2522-schema.sql
+			db%22.t%2522.0.csv
+	*/
+
+	s.touch(c, "test-schema-create.sql")
+	s.touch(c, "test.t%22-schema.sql")
+	s.touch(c, "test.t%22.sql")
+	s.touch(c, "test.t%2522-schema.sql")
+	s.touch(c, "test.t%2522.csv")
```

Comment:
include the following two cases:

* `test.t%gg.csv`
* `test.t+gg.csv` (with this you should realize that you probably shouldn't use `url.QueryUnescape`)

Reply:
done

Reply:
Replace with `url.PathUnescape`😅